loader.c

```diff
@@ -117,7 +118,17 @@ static int prot_from_phdr(const ElfW(Phdr) *phdr)
         prot |= PROT_WRITE;
     if (phdr->p_flags & PF_X)
         prot |= PROT_EXEC;
+    /*
+     * FIXME
+     * In ARM, some global variables in .text will appear in .rel.dyn,
+     * and we need to give it WRITE permission to relocate it.
+     * It must be fixed due to security issue !!!
```

Comment:
Would you like to fix as well?

Reply:
OK I'll try it.
but it better to open an new issue